oxen-rust/test-put-demo.sh

```
@@ -0,0 +1,477 @@
+    echo "  workdir_name: Optional name for working directory (default: oxen-put-demo)"
+    exit 1
+fi
+
+OXEN_TARGET_DIR="$1"
+WORKDIR="${2:-oxen-put-demo}"
+OXEN_CLIENT="$OXEN_TARGET_DIR/oxen"
+OXEN_SERVER="$OXEN_TARGET_DIR/oxen-server"
+
+pkill oxen-server
+rm -rf eeee/oxen-put-demo/
+cd eeee
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)